index.html

```diff
@@ -40,5 +61,361 @@ <h2>
+                  </li>
+                </ol>
+              </li>
+              <li>
+                <p>Return <var>p</var>.</p>
+              </li>
+            </ol>
+            <p>
+              The <a>user agent</a> MUST NOT capture content that's behind a
+              partially transparent captured
+              <a data-cite="SCREEN-CAPTURE#dfn-display-surface">display surface</a>.
+            </p>
+            <p>
+              The <a>user agent</a> MUST NOT share the audio other than audio emitted
+              from the captured tab, and MUST NOT share audio of the entire system.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h2 id="constrainable-properties">
+          Constrainable Properties for Captured Viewport Surfaces
+        </h2>
+        <p>The constraints relevant to {{MediaDevices/getViewportMedia}} are only
+        those relevant to {{MediaDevices/getDisplayMedia()}}, as defined in
+        <a data-cite="SCREEN-CAPTURE#constrainable-properties">
+          5.4 Constrainable Properties for Captured Display Surfaces</a>.
+      </section>
+      <section data-cite="permissions">
+        <h1 id="permissions-intergration">Permissions Integration</h1>
+        <p>
+          <cite>Viewport Capture</cite> is a [=powerful feature=] which is identified by the
+          [=powerful feature/name=] "viewport-capture", requiring [=express permission=] to be used.
+        </p>
+        <p>
+          As required for integration with the [[[Permissions]]] specification, this specification defines the following:
+        </p>
+        <dl>
+          <dt>
+            [=powerful feature/permission state constraints=]
+          </dt>
+          <dd>
+            Valid values for this descriptor's <a>permission state</a> are
+            {{PermissionState/"prompt"}} and {{PermissionState/"denied"}}. The user agent MUST NOT
+            ever set this descriptor's <a>permission state</a> to {{PermissionState/"granted"}}.
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h1 id=feature-policy-integration>Permissions Policy Integration</h1>
+        <p>This specification defines a [=policy-controlled feature=]
+        identified by the string
+        <code><dfn data-lt="display-capture-feature">"viewport-capture"</dfn></code>.
+        Its [=default allowlist=] is <code>"self"</code>.
+        <div class="note">
+          <p>A [=document=]'s
+          [=Document/permissions policy=]
+          determines whether any content in that document is allowed to use
+          {{MediaDevices/getViewportMedia}}. If disabled in any document, no content in
+          the document will be
+          [=allowed to use=]
+          {{MediaDevices/getViewportMedia}}. This is enforced by the
+          [=prompt the user to choose=] algorithm.
+          </p>
+        </div>
+      </section>
+    </section>
+    <section>
+      <h1>Privacy Indicator Requirements</h1>
```

Comment:
```suggestion
      <h1>Privacy considerations</h1>
```

Reply:
Shouldn't we keep the naming consistent with:
1. [Privacy Indicator Requirements](https://w3c.github.io/mediacapture-main/#privacy-indicator-requirements) in getUserMedia, and
1. [Privacy Indicator Requirements](https://w3c.github.io/mediacapture-screen-share/#privacy-indicator-requirements) in getDisplayMedia

?

Reply:
that makes sense; I guess we do need a separate Privacy Considerations section though.

Reply:
I've renamed the TBD section "Security and Privacy Considerations" for now (the camel case is for consistency with the other headings in this doc).